ChessBoard/ChessBoard.cs

```diff
@@ -0,0 +1,40 @@
+namespace ChessBoardAssigment
+{
+    public class ChessBoard
+    {
+        public ChessBoard() { }
+
+        public void GenerateChessBoardOneLoop(int numberOfRows)
+        {
+            Console.WriteLine(
+                "Chessboard where 1 represents white square and 0 represents black square:"
```

Comment:
I'd suggest separating responsibilities - GenerateChessBoardOneLoop method should return only the generated board as a string or some similar structure like array of strings (each line a separate element). 
The other text like "Chessboard wher..." could be considered to be a part of dialog ;)